bindings/jupyter-modules/jupyter-ma-causal/src/containers/slidebar-svg-container/index.js

```diff
@@ -44,19 +44,20 @@ class Chart extends Component {
   }
 
   getLeftLabel() {
+    const format = d3Format('.6f');
     const {sliderValue, data, lineName} = this.props;
     if (sliderValue === null || sliderValue === undefined) {
-      return '1.0';
+      return format(data[data.length - 1].y);
```

Comment:
please add sanity checks for data (null and isArray) is we were to call `.length`

Reply:
Data is assumed to be an array time, as specified in the prop-types of the component. However,  data can be an empty array, so added the check for empty array.